Gemfile

```diff
@@ -11,38 +13,51 @@ def location_for(place, fake_version = nil)
 end
 
 group :system_tests do
-  gem 'rake'
-  gem 'beaker-rspec', *location_for(ENV['BEAKER_RSPEC_VERSION'] || '~> 6.0')
-  gem 'beaker-pe', *location_for(ENV['BEAKER_PE_VERSION'] || '~> 1.1')
-  gem 'listen', '<3.1.0'
+  gem 'rake',                    '~> 0'
+  gem 'beaker-rspec',            *location_for(ENV['BEAKER_RSPEC_VERSION'] || '<  5.7') if RUBY_VERSION <  '2.1.5'
```

Comment:
The `dup` is to handle a Bundler bug - https://github.com/bundler/bundler/issues/3187

https://github.com/puppetlabs/puppet/commit/fb9995da521269c17755c15dcb2d19afa7932501